components/fibery/sources/entity-created/entity-created.mjs

```diff
@@ -0,0 +1,21 @@
+import base from "../common/webhooks.mjs";
+
+export default {
+  ...base,
+  key: "fibery-entity-created",
+  name: "New Entity Created",
+  description: "Emit new event for every created entity of a certain type",
+  type: "source",
+  version: "0.0.1",
+  async run(event) {
+    console.log(`Received new event with ${event.body.effects.length} sequence(s)`);
+    event.body.effects
+      .filter(({ effect }) => effect === "fibery.entity/create")
+      .forEach((effect) => {
+        this.$emit(effect, {
+          summary: `New created entity: ${effect.id}`,
```

Comment:
you could add the `id` of the entity on the $emit function
`this.$emit(effect, {
          id: effect.id,
          summary: `New created entity: ${effect.id}`,`